modules/installing-trustyai-service.adoc

```diff
@@ -0,0 +1,23 @@
+:_module-type: CONCEPT
+
+[id='installing-trustyai-service_{context}']
+= Installing the TrustyAI service for a data science project
+
+[role='_abstract']
+
+Install TrustyAI into your project and provides access to its features for all models deployed within that project so that data scientists can monitor models. An instance of the TrustyAI service is required for each data science project, or namespace, that contains models that your team wants to monitor. 
```

Comment:
```suggestion
Install TrustyAI into your project and provide access to its features for all models deployed within that project so that data scientists can monitor models. An instance of the TrustyAI service is required for each data science project, or namespace, that contains models that your team wants to monitor. 
```
Minor typo - might have been my suggestion!

Reply:
I think I can delete that comma after namespace too?